backend/BackendMIIT/src/main/java/com/example/BackendMIIT/model/dto/ProfileDto.java

```diff
@@ -1,9 +1,11 @@
 package com.example.BackendMIIT.model.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
What the purpose of this annotation? It will work if you remove it

Reply:
No, unfortunately, the null fields are showing without this annotation:
![image](https://github.com/user-attachments/assets/a3e49a4a-04b4-4f26-ac2e-94d149859b98)